### PR TITLE
Update what Rackspace is used for

### PR DESCRIPTION
**Pull request by ekohl**

### CODE REVIEW

File: docs/index.md [hunk 1]
```diff
@@ -33,7 +33,7 @@ A list of the hosting we have, who provides it, and what capacity it has
 
 ### Rackspace
   * Previously sponsored, [being phased out](https://community.theforeman.org/t/rfc-moving-off-of-rackspace-infrastructure/17932)
-  * Hosts Jenkins and Foreman
+  * Used for Mailgun (to send email from Redmine and Discourse) and CloudFiles (to store Fastly CDN logs)
```
Comment (evgeni) [inline docs/index.md:36]: The later is not correct anymore, @ares deleted the logs

**Review by evgeni — CHANGES_REQUESTED** [review]
Should we also update the places where the services are now hosted?

File: docs/index.md (hunk 1 above)
Comment (evgeni) [reply docs/index.md:36]: the first one: yes

the second: why?

Comment (ekohl) [reply docs/index.md:36]: I may be tired and somehow remembered web02, but that was web02.rackspace.

**Comment by ekohl** [comment]
> Should we also update the places where the services are now hosted?

I'm not sure what you mean by this.

Otherwise updated.

**Review by evgeni — APPROVED** [review]

**Comment by evgeni** [comment]
> > Should we also update the places where the services are now hosted?
> 
> I'm not sure what you mean by this.

Like: Jenkins server and Puppet server and Foreman are now at Conova, right?

**Pull request merged**